### PR TITLE
Improve messages of existing errors in SCM/CI integration

### PR DESCRIPTION
**Pull request by dmarcoux**
With the changes in this PR, we can drop some errors from the `Errors` table in the user documentation of the SCM/CI integration. This is happening in the PR https://github.com/openSUSE/obs-docu/pull/235.

### CODE REVIEW

**Comment by dmarcoux** [comment]
[Improve error messages (M)](https://trello.com/c/59d7gK6G/1868-improve-error-messages-m)

File: src/api/app/validators/workflow_filters_validator.rb [hunk 1]
```diff
@@ -1,11 +1,19 @@
 class WorkflowFiltersValidator < ActiveModel::Validator
   SUPPORTED_FILTER_VALUES = [:only, :ignore].freeze
+  DOCUMENTATION_LINK = 'https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html' \
+                       '#sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters'.freeze
```
Comment (rubhanazeem) [inline src/api/app/validators/workflow_filters_validator.rb:4]: Do you think we can break this link into two parts? For example 
```
DOCUMENTATION_BASE_LINK = 'https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html'
```
And then define the link path:
```
DOCUMENTATION_PATH = '#sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters'
```
Reason is `DOCUMENTATION_BASE_LINK` is used in few places above and it's common. We don't have to hard code the `DOCUMENTATION_BASE_LINK` in those places.

And do you think we can move `DOCUMENTATION_BASE_LINK`  and `DOCUMENTATION_PATH` to `src/api/app/models/token/workflow.rb` ?

Comment (dmarcoux) [reply src/api/app/validators/workflow_filters_validator.rb:4]: I prefer to keep this simple. For now, we only have twice a link to the documentation. If this grows in the future, we can change it.

**Review by rubhanazeem — DISMISSED** [review]

File: src/api/app/validators/workflow_filters_validator.rb (hunk 1 above)
Comment (dmarcoux) [reply src/api/app/validators/workflow_filters_validator.rb:4]: I changed my mind, I'll do it.

**Comment by dmarcoux** [comment]
**DO NOT MERGE**: I'll apply what was discussed [here](https://github.com/openSUSE/open-build-service/pull/12412#discussion_r844996566)

**Review by rubhanazeem — APPROVED** [review]

**Pull request merged**